worldedit-core/src/main/java/com/sk89q/worldedit/command/SchematicCommands.java

```diff
@@ -325,8 +328,9 @@ public void load(
         LocalConfiguration config = worldEdit.getConfiguration();
 
         //FAWE start
+        final Closer closer = Closer.create();
```

Comment:
Can this be moved into  the outer try as a resource? Then the closing doesn't need to be handled explicitly